spec/src/main/resources/META-INF/smithy/bsp/extensions/rust.smithy

```diff
@@ -0,0 +1,325 @@
+@unstable
+@jsonRequest("buildTarget/rustWorkspace")
+operation RustWorkspace {
+    input: RustWorkspaceParams
+    output: RustWorkspaceResult
+}
+
+@unstable
+structure RustWorkspaceParams {
+    /// A sequence of build targets for workspace resolution.
+    @required
+    targets: BuildTargetIdentifiers
+}
+
+@unstable
+structure RustWorkspaceResult {
+    /// Packages of given targets.
+    @required
+    packages: RustPackages
+    /// Dependencies in `cargo metadata` as listed in the package `Cargo.toml`,
+    /// without package resolution or any additional data.
+    @required
+    rawDependencies: RustRawDependencies
+    /// Resolved dependencies of the package. Handles renamed dependencies.
+    /// Correspond to dependencies from resolved dependency graph from `cargo metadata` that shows
+    /// the actual dependencies that are being used in the build.
+    @required
+    dependencies: RustDependencies
+    /// A sequence of build targets taken into consideration during build process.
+    @required
+    resolvedTargets: BuildTargetIdentifiers
+}
+
+list RustPackages {
+    member: RustPackage
+}
+
+/// A `crate` is the smallest amount of code that the Rust compiler considers at a time.
+/// It can come in one of two forms: a binary crate or a library crate.
+/// `Binary crates` are programs you can compile to an executable that you can run,
+/// such as a command-line program or a server.
+/// Each must have a function called main that defines what happens when the executable runs.
+/// `Library crates` don’t have a main function, and they don’t compile to an executable.
+/// Instead, they define functionality intended to be shared with multiple projects.
+///
+/// A `package` is a bundle of one or more crates that provides a set of functionality.
+/// It contains a Cargo.toml file that describes how to build those crates.
+/// A package can contain many binary crates, but at most only one library crate.
+/// However, it must contain at least one crate, whether that’s a library or binary crate.
+structure RustPackage {
+    /// The package’s unique identifier
+    @required
+    id: String
+    /// The package's root path.
+    @required
+    rootUrl: URI
+    /// The name of the package.
+    @required
+    name: String
+    /// The version of the package.
+    @required
+    version: String
+    /// Defines a reason a package is in a project.
+    @required
+    origin: RustPackageOrigin
+    /// Code edition of the package.
+    @required
+    edition: RustEdition
+    /// The source ID of the dependency, for example:
+    /// "registry+https://github.com/rust-lang/crates.io-index".
+    /// `null` for the root package and path dependencies.
+    source: String
+    /// Corresponds to source files which can be compiled into a crate from this package.
+    /// Contains only resolved targets without conflicts.
+    @required
+    resolvedTargets: RustTargets
+    /// Same as `targets`, but contains all targets from this package.
```

Comment:
What does `targets` refer to here? Is it the `targets` in `RustWorkspaceParams`.

Reply:
I changed 'targets' to 'resolvedTargets' but I forgot to change the name in the comment. I fixed it to 'resolvedTargets`